index.html

```diff
@@ -42,5 +46,265 @@ <h1 class=" bigger-text">I'm Albert. Glad to see you!</h1>
+              HTML
+            </li>
+            <li class="tag">
+              CSS
+            </li>
+            <li class="tag javascript">
+              JavaScript
+            </li>
+          </ul>
+          </div>
+          <div class="see-project">
+            <button class="see-project-button" type="submit">
+              See project
+            </button>
+          </div>
+          </div>
+          </div>
+        </div> 
+    </section>
+    <section>
+      <div class="about-container">
+          <div class="about-top">
+              <div>
+                  <h1>
+                      About Myself
+                  </h1>
+              </div>
+              <div>
+                  <p class="about-text">
+                      Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                      If you like what you see and have a project you need coded, don't hestiate to contact me.
+                  </p>
+              </div>
+              <div>
+                  <div class="connect">
+                      LET'S CONNECT
+                  </div>
+                  <div>
+                      <ul class="social">
+                          <li>
+                              <a href="#"><img src="Vector@2x.png" alt="Vector" width="17px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="Linkedin icon@2x.png" alt="Linkedin" width="20px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="vecto@2x.png" alt="vecto" width="14px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="twitter.png" alt="twitter" width="19.99px" height="19.99px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="media.png" alt="media" width="20px" height="16px"></a>
+                          </li>
+                      </ul>
+                  </div>
+              </div>
+              <div>
+                <button class="see-project-button" type="button">Get my resume</button>
+              </div>
+              <ul class="about-list">
+                <li class="tab-item">
+                    <div class="language-title"><span class="tab-text language-tittle">Languages <i class='fas fast'>&#xf078;</i></span></div>
+                    <ul class="tab-langs">
+                        <div class="frame-lang frame"><img src="js.png"><span class="tab-text">JavaScript</span></div>
+                        <div class="frame-lang"><img src="html.png"><span class="tab-text">HTML</span></div>
+                        <div class="frame-lang"><img src="css.png"><span class="tab-text">CSS</span></div>
+                    </ul>
```

Comment:
- [ ] Instead of using `div` tags for the list items inside the `ul` list container I would suggest using `li` tags as it would be semantically more appropriate. `divs` mean little on their own, whereas `ul ``lis` explicitly mean "this is an un/ordered list of items." The term "semantics" refers to the way that you use the inherent meaning of existing structures to create explicit meaning.